pkg/docker/docker.go

```diff
@@ -132,6 +132,55 @@ type RunContainerOptions struct {
 	CapDrop          []string
 }
 
+// asDockerConfig converts a RunContainerOptions into a Config understood by the
+// docker client library.
```

Comment:
that's right, thanks. I had in mind "go-dockerclient", therefore wrote in lowercase. Should be either `go-dockerclient` or `Docker client`. I prefer the former as it's more specific to what I mean. Thanks for pointing it out!